### ISSUE TITLE
FR: Blog Collage Block (3)

### ISSUE THREAD

**Issue by shirin27** [hlxsites/vg-macktrucks-com#602]
From vg-macktrucks-com-rd created by [Gonzalo-CG](https://github.com/Gonzalo-CG): Netcentric/vg-macktrucks-com-rd#131  

### Description
The [Bulldog Magazine Blog page](https://www.figma.com/file/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2210%3A65358&mode=dev) has several blog categories for which a feed of blogs has to be displayed.

The feed of blogs has the same design and structure as the default variation of #600, For use on the category page. **This issue will only tackle the initial load of the first 9 blog entries**. Sorting and the "Show More" functionality will be developed on issue https://github.com/hlxsites/vg-macktrucks-com/issues/606

### Acceptance Criteria
- [x] The Blog Feed Block is available for authors to use on any page.
- [x] The Blog Feed Block has the same design for Desktop & mobile as shown in Figma.
- [x] On first load, 9 blog entries are displayed.
- [x] Logic is introduced so that the block can automatically update when new articles are published to always display the most recent articles without requiring manual interventions from editors.
- [x]  Articles appear in descending order based on their publish date, with the most recent articles appearing first.
- [x] The block will be named v2-explore-articles 

### Related Ticket
https://github.com/hlxsites/vg-volvotrucks-us/issues/680

### Figma
The design for this block is the following ([Figma link](https://www.figma.com/design/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=15073-47376&m=dev)):
<img src="https://github.com/Netcentric/vg-macktrucks-com-rd/assets/139977780/136a73b1-13bf-46b4-9d85-b8af8ec14439" width="50%">

###  Questions
- What is the initial amount of loaded blogs? **9**
- How many blogs are loaded on Show More press? **An additional block of 9**
- What are the sorting options? **Date (most recent first) and Title (alphabetical ascending order)** - this will be handled in a separate ticket.

### Out of scope
- Only blogs of the current category are loaded. but instead of showing a static amount of elements this block renders an initial amount and has a "Show More" functionality.
- Videos will be handled in a [separate ticket ](https://github.com/hlxsites/vg-macktrucks-com/issues/805)
-  The block showing a message showing the current amount of loaded articles and the total amount of articles, and a dropdown to allow sorting will be handled in a separate ticket.

**Comment by shirin27** [comment]
@WendyKruger Please see "Open Questions" above.

**Comment by shirin27** [comment]
### Open Questions

- What is the initial amount of loaded blogs? **9**
- How many blogs are loaded on Show More press? **An additional block of 9**
- What are the sorting options? **Date (most recent first) and Title (alphabetical ascending order)**

**Comment by mollyaljenkins** [comment]
Blocked by #600

**Comment by alexiscoelho** [comment]
This block will instead be called v2-explore-articles

@ZoeBeykirch  do you agree with this?

**Comment by alexiscoelho** [comment]
Also @ZoeBeykirch 
1 - The articles don't have videos so we can show them on the grid. If we need this, we will have to think about it and implement it in a different US.
2 - Can we extract the sorting feature to another Ticket?
3 - Could this be a component to use in any page and show all categories and if in a category page it will just show articles from that category?

**Comment by alexiscoelho** [comment]
Extra tickets to be created that are out of scope on this ticket:
1 - Video
2 - Sorting feature
3 - Filter by category in a category page
@ZoeBeykirch

**Comment by girishdigrajkar** [comment]
@jonatan-lledo-netcentric , @alexiscoelho : Validated and below are few outstanding issues :
Except for below cosmetic observations all functionality works fine :

**Desktop:**
1.  First column image should be  720px X 540px whereas, It it 520 X 390;        [GD] Fixed now 
2. Second column image should be 720px X 270px whereas it is  520pxX 195px   [GD] Fixed now 
3.  As above 2 issues all 9 cards sizing differs    [GD] Fixed now 
4. Category title on article card has font 13px whereas, it should be 12px    [GD] Fixed now 
5.  **Show more button size should be  160px X 47.5px  however, in actual it is  271px X 48p** 
 

**Mobile:**
1. Category title on article card has font 13px whereas, it should be 12px [GD] Fixed now 
**2. Article card size should be 375pxX280px whereas, it is 375pxX312.5px (iPhone)  and  for android it is 375pxX300 px**
**3. Show more button size should be  271px X 48px whereas, it is  311px X 47.5px (iPhone) and  296px X 47.5px (Android)**

**Comment by girishdigrajkar** [comment]
Validated and based on dev inputs for outstanding issue we can proceed closing this ticket.
